BraveWallet/Crypto/OriginText.swift

```diff
@@ -5,22 +5,27 @@
 
 import BraveCore
 import SwiftUI
+import struct Shared.Strings
```

Comment:
Don't need to import Shared anymore post-SPM (`import Strings` should be enough)